js/app/models/simpleEventModel.js

```diff
@@ -41,7 +41,8 @@ app.factory('SimpleEvent', function () {
 		'repeating': null,
 		'rdate': null,
 		'rrule': null,
-		'exdate': null
+		'exdate': null,
+		'spreedmeeting': null
```

Comment:
`spreedmeeting` is not part of RFC5545 ;)
Please use `X-SPREED-MEETING` as noted earlier already